### DISCUSSION TITLE
ExpressLRS Restore Firmware to earlier version

### DISCUSSION THREAD

**Discussion by simpsmf** [ExpressLRS/ExpressLRS#2092] [General]
I am 3.1.2 on my RadioMaster TX16s transmitter. I want to go back to 2.5.2. Every time I flash to 2.5.2, or any version in that range, it reverts back to 3.1.2 on it's own. The reason I want to do this is that I can't seem to get an ER5A reciever to bind with the radio except in the 2.5 version range. I was binding at the 3.2.0 version, but it quit abruptly. I managed to get it to bind in the 2.5 range, but when I cycle power on my radio, it reverts back to the 3.?? range. I have the same version of firmware on my ER5A reciever as my radio each time I try to bind. I am using the passphrase system. I have only had this radio for a couple of weeks. 
How can I flash back from 3.xx to 2.xx and have it stay when I cycle power on my radio.

**Comment by sercona** [comment]
reverts back on its own?  can you explain that?

I have the er5a and it works fine in 3.x elrs.  my tx's are all radiomaster with various tx modules, all at 3.x (I have nothing that cant be upgraded to 3.x).

if something quit, maybe there is another issue, but if you flashed and were able to bind at 3.x with tx and rx and the bind phrase stuff is all setup ok, the problem sounds like it could be elsewhere, maybe.
